File: ungs-docsys-bff/src/modules/resume-user/dtos/contact-response.dto.ts
import { ApiProperty } from '@nestjs/swagger';

export class ContactResponseDto {
  @ApiProperty({ example: 'user@example.com' })
  email: string;

  @ApiProperty({ example: '+1234567890' })
  phone: string;

  @ApiProperty({ example: '123 Main St, City, Country' })
  address: string;

  @ApiProperty({ example: 'https://linkedin.com/in/username' })
  linkedin: string;
}

File: ungs-docsys-ms/src/main/java/com/ungs/docsys/dtos/ContactResponseDto.java
package com.ungs.docsys.dtos;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ContactResponseDto {
    private String email;
    private String phone;
    private String address;
    private String linkedin;
}
